### DISCUSSION TITLE
My PHP Problem after running  photon

### DISCUSSION THREAD

**Discussion by azzahrah** [komoot/photon#570] [Q&A]
[Production Environment]

Distributor ID: Ubuntu
Description:    Ubuntu 16.04.5 LTS
Release:        16.04
Codename:       xenial
VPS Conbabo Spect
8 vCPU Cores
30 GB RAM
800 GB SSD

photon running well with indonesia map imported from nominatim

```
java -jar -Xmx8192m photon-0.3.4.jar
```
before start photon  we can get mysql data more than 500 row from php script, after running photon we only get data 100row?

belo is my top information

```

%Cpu(s):  5.9 us,  3.0 sy,  0.0 ni, 85.6 id,  0.9 wa,  0.0 hi,  0.9 si,  3.6 st
KiB Mem : 30815092 total, 25781264 free,  2924688 used,  2109140 buff/cache
KiB Swap:        0 total,        0 free,        0 used. 27140980 avail Mem

```

**Comment by lonvia** [comment]
Please use the discussion section for questions.
